File: utils/prompts.py
from shutil import which

from shellrunner import X


def reboot():
    response = input("Reboot? (y/N) ")
    if response.lower() == "y":
        if which("wsl.exe"):
            print("WSL detected. Shutting down...")
            X("powershell.exe -Command wsl --shutdown")
        else:
            X("sudo reboot")
    return False


def continuep():
    response = input("Continue? (Y/n) ")
    return response.lower() != "n"

File: utils/prompts.ts
import { $ } from "bun"
import { getInput } from "./helpers.ts"

async function reboot() {
  const response = await getInput("Reboot? [y/N] ")
  if (response.toLowerCase() === "y") {
    await $`sudo reboot`
    return true
  }
  return false
}

async function continuep() {
  const response = await getInput("Continue? [Y/n] ")
  return response.toLowerCase() !== "n"
}

export { continuep, reboot }
